base/src/test/java/io/spine/logging/LoggingTest.java

```diff
@@ -20,27 +20,93 @@
+import com.google.common.flogger.FluentLogger;
+import com.google.common.flogger.LogContext;
+import com.google.common.flogger.LoggerConfig;
+import com.google.common.flogger.backend.LogData;
 import io.spine.logging.given.LoggingObject;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
-import org.slf4j.Logger;
-import org.slf4j.helpers.SubstituteLogger;
+
+import java.util.function.Supplier;
+import java.util.logging.Level;
 
 import static io.spine.testing.logging.LogTruth.assertThat;
 
 @DisplayName("Logging interface should")
 class LoggingTest {
 
     @Test
-    @DisplayName("obtain Logger instance")
-    void loggerInstance() {
+    @DisplayName("assume `Level.FINE` for debug")
+    void debugLevel() {
+        assertThat(Logging.debugLevel())
+             .isEqualTo(Level.FINE);
+    }
+
+    @Test
+    @DisplayName("assume `Level.SEVER` for errors")
+    void errorLevel() {
+        assertThat(Logging.errorLevel())
+                .isEqualTo(Level.SEVERE);
+    }
+
+    @Test
+    @DisplayName("obtain same `FluentLogger` Instance")
+    void fluentLogger() {
         Logging object = new LoggingObject();
-        Logger logger = object.log();
-        Subject<DefaultSubject, Object> assertLogger = assertThat(logger);
 
-        assertLogger.isNotNull();
-        assertLogger.isInstanceOf(SubstituteLogger.class);
+        FluentLogger logger = object.logger();
+
+        assertThat(object.logger())
+             .isSameInstanceAs(logger);
+        assertThat(new LoggingObject().logger())
+             .isSameInstanceAs(logger);
+    }
+
+    @Nested
+    @DisplayName("expose shortcut methods")
+    class Shortcuts {
+
+        private Logging object;
+        private FluentLogger logger;
+
+        @BeforeEach
+        void createLoggingObject() {
+            object = new LoggingObject();
+            logger = object.logger();
+        }
+
+        @Test
+        @DisplayName("for standard logging levels")
+        void shortcutMethods() {
+            assertApi(object::_config, Level.CONFIG);
+            assertApi(object::_fine, Level.FINE);
+            assertApi(object::_finer, Level.FINER);
+            assertApi(object::_finest, Level.FINEST);
+            assertApi(object::_info, Level.INFO);
+            assertApi(object::_severe, Level.SEVERE);
+            assertApi(object::_warn, Level.WARNING);
+        }
+
+        @Test
+        @DisplayName("for popular aliases")
+        void aliases() {
+            assertApi(object::_debug, Level.FINE);
+            assertApi(object::_trace, Level.FINEST);
+            assertApi(object::_error, Level.SEVERE);
+        }
+
+        private void assertApi(Supplier<FluentLogger.Api> method, Level expectedLevel) {
+            LoggerConfig.of(logger)
+                        .setLevel(expectedLevel);
+
+            FluentLogger.Api api = method.get();
+            assertThat(api)
```

Comment:
Why do you place `isInstanceOf` to the next line?

Reply:
I usually put asserting methods on the next line. This way they are at the constant offset at the source code, and it's faster to read the code.  You don't have to travel left-to-right with your eyes.